docker-compose.yml

```diff
@@ -39,3 +39,11 @@ services:
     ports:
       - "127.0.0.1:8999:3306"
     container_name: mysql-57
+
+  oracle-12c:
+    image: konnecteam/oracle12c-database
+    volumes:
+      - ./oracle-data:/u01/app/oracle
```

Comment:
this mapping can be removed when you decide to use the version I proposed to your dockerfile :)